concordium-node/src/bin/collector.rs

```diff
@@ -162,11 +146,8 @@ async fn main() {
         thread::sleep(Duration::from_millis(conf.artificial_start_delay));
```

Comment:
I know this is from before already, but I think because we use `tokio`, it would be better to use the [`tokio::sleep`](https://docs.rs/tokio/latest/tokio/time/fn.sleep.html) instead of thread::sleep. The former works better in combination with async futures.